packages/o365_metrics/data_stream/teams_device_usage_user_counts/manifest.yml

```
@@ -0,0 +1,56 @@
+title: Microsoft Office 365 Teams Device Usage User Counts metrics
+type: metrics
+streams:
+  - input: cel
+    title: Office 365 Teams Device Usage User Counts metrics.
+    enabled: true
+    description: Collect Office 365 Teams Device Usage User Counts metrics.
+    template_path: cel.yml.hbs
+    vars:
+      - name: interval
+        type: text
+        title: Interval
+        description: The interval at which the API is polled, supported in seconds, minutes, and hours.
```

As per https://github.com/elastic/obs-integration-team/issues/156

```suggestion
        description: Interval of API polling. Interval supported in seconds, minutes and hours.
```

---

I would prefer to stick with what's already in place.